examples/logger.py

```diff
@@ -186,10 +186,10 @@ def receivedSlot(self, channelName: str, content: Any):
             if isinstance(content, str):
                 self.addLog(content)
             else:
-                print("The message for the channel log should be a string.")
+                self.logger.info("The message for the channel log should be a string.")
```

Comment:
Please use `logger = logging.getLogger(__name__)` not `self.logger`!

Reply:
I remember I heard that we should not set logger as global variable. But I will set it as global variable for now.

Reply:
Can you explain this more?

Reply:
I changed my previous plan using global variable as below. 
1. In constructor of LoggerApp, set root logger as local variable and set its level as "DEBUG" and addHandler.
2. When we need logger of logger.py (i.e. root logger), just call root logger using logging.getLogger(). (line 189, 191)

Reply:
You can use multiple loggers. The root logger is only for _reading_ the logs.
To _write_ logs in `loggerApp`, please use `logging.getLogger(__name__)`.

Reply:
I also agree with @kangz12345's suggestion!